ibmcloudshellv1/ibm_cloud_shell_v1_integration_test.go

```diff
@@ -152,10 +152,6 @@ var _ = Describe(`IBMCloudShellV1 Integration Tests`, func() {
 					Enabled: core.BoolPtr(true),
 					Key:     core.StringPtr("eu-de"),
 				},
-				{
-					Enabled: core.BoolPtr(false),
-					Key:     core.StringPtr("jp-tok"),
-				},
```

Comment:
Do you intend to remove these? Just asking because as far as I can see, the API hasn't changed.

Reply:
Yes , we have deprecated the tokyo region.